tests/models/test_dag.py

```diff
@@ -1848,9 +1888,6 @@ def test_dags_needing_dagruns_not_too_early(self):
         dag_models = DagModel.dags_needing_dagruns(session).all()
         assert dag_models == []
 
-        session.rollback()
-        session.close()
```

Comment:
Removed this because of the added clean_db method

Reply:
But with these rollbacks it _shouldn't_ need a clean method.

Reply:
Yes. I was seeing that it was repeated, happy to return it back if it's preferred